go-controller/pkg/ovn/service.go

```diff
@@ -512,3 +512,27 @@ func svcQualifiesForReject(service *kapi.Service) bool {
 	_, ok := service.Annotations[OvnServiceIdledAt]
 	return !(config.Kubernetes.OVNEmptyLbEvents && ok)
 }
+
+// SVC can be of types 1. clusterIP, 2. NodePort, 3. LoadBalancer,
+// or 4.ExternalName In this scenario we will only address 1.
```

Comment:
4. is ExternalIP

Why are we limiting this to just the clusterIP? Why not just return all of the potential vips for cluster ip, node ip, external ip and ingress ip? The logic to parse these is all over the service.go file. Cant we just copy it?